tensorboard/webapp/runs/views/runs_table/sorting_utils_test.ts

```diff
@@ -28,6 +28,7 @@ describe('sorting utils', () => {
       expect(parseNumericPrefix('0')).toEqual(0);
       expect(parseNumericPrefix('123')).toEqual(123);
       expect(parseNumericPrefix('123/')).toEqual(123);
+      expect(parseNumericPrefix('123/foo')).toEqual(123);
```

Comment:
This still has a '/' after the number. I was thinking something like 123train. Still just a nit though.

Reply:
Ahh, okay. I'll add that case too